03/app.js

```diff
@@ -5,13 +5,80 @@ const a = randomNumber(min, max);
 const b = randomNumber(min, max);
 const c = randomNumber(min, max);
 
-console.log(a, b, c);
+const sumNumbers = getSum(4, 25, 14)
+// console.log(sumNumbers);
 
+function getSum(a, b, c) {
+    const convertedA = parseInt(a);
+    const convertedB = parseInt(b);
+    const convertedC = parseInt(c); 
 
+    if (convertedA>=convertedB && convertedA>convertedC && convertedB>=convertedC) {
+        return convertedA+convertedB;
+    }
+    else if (convertedA>convertedB && convertedA>=convertedC && convertedC>=convertedB) {
+        return convertedA+convertedC;
+    }
+    else if (convertedB>convertedA && convertedB>=convertedC && convertedC>=convertedA) {
+        return convertedB+convertedC;
+    }
+    else if (convertedB>=convertedA && convertedB>convertedC && convertedA>=convertedC) {
+        return convertedB+convertedA;
+    }
+    else if (convertedC>=convertedA && convertedC>convertedB && convertedA>=convertedB) {
+        return convertedC+convertedA;
+    }
+    else if (convertedC>convertedA && convertedC>=convertedB && convertedB>=convertedA) {
+        return convertedC+convertedB
+    }
+    else {
+        console.log('Wszystkie cyfry są takie same');
+    }
```

Comment:
hmm... czy jeśli mielibyśmy 10 liczb to też byś tak to zrobił ;P Może da się te liczby wrzucić do tablicy i posortować?